appinventor/components-ios/src/Label.swift

```diff
@@ -29,6 +34,25 @@ public final class Label: ViewComponent, AbstractMethodsForViewComponent {
     parent.add(self)
     Height = kLengthPreferred
     Width = kLengthPreferred
+    FontSize = Float64(kFontSizeDefault)
+  }
+  
+  @objc func updateFontSize() {
```

Comment:
This does not need to be marked as `@objc` as it won't be called directly from Scheme.